### PR TITLE
[Bug](bloomfilter) Fix wrong result using bloomfilter with date type

### PR DESCRIPTION
**Pull request by Gabriel39**
# Proposed changes

before:
image
after:
image

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Gabriel39** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 33.78 seconds
 stream load tsv:          480 seconds loaded 74807831229 Bytes, about 148 MB/s
 stream load json:         38 seconds loaded 2358488459 Bytes, about 59 MB/s
 stream load orc:          69 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230228122353_clickbench_pr_105714.html

File: be/src/exprs/bloom_filter_func.h [hunk 1]
```diff
@@ -341,6 +379,44 @@ struct DateFindOp : public CommonFindOp<DateTimeValue> {
     void insert(BloomFilterAdaptor& bloom_filter, const void* data) const {
         bloom_filter.add_bytes((char*)data, sizeof(vectorized::VecDateTimeValue));
     }
+
```
Comment (HappenLee) [inline be/src/exprs/bloom_filter_func.h:382]: why directly use parent class `CommonFindOp<VecDateTimeValue>` not we do not have old column exec engine.

**Comment by yiguolei** [comment]
Should add a regression test.

**Comment by Gabriel39** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**